Update to abscissa 0.6.0-beta.1 and clap 3.0.0-beta.5 (#1576)
informalsystems/ibc-rs: guide/src/help.md

    --port-a PORT-A           identifier of the side `a` port for the new channel
    --port-b PORT-B           identifier of the side `b` port for the new channel
    -o, --order ORDER         the channel ordering, valid options 'unordered' (default) and 'ordered'

>     -v, --channel-version VERSION     the version for the new channel

Reviewer: ```suggestion
    -v, --version VERSION     the version for the new channel
```

```

> Additionally, the `-h`/`--help` flags typical for CLI applications work on
> most commands (with the exception of some commands currently using `-h` for
> other purposes; these should be changed to follow the convention in a
> future version).

Reviewer: ```suggestion
The `help` command is a replacement of the familiar `-h`/ `--help` flag typical for CLI applications.
```
